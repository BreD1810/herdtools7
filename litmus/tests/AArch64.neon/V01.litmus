AArch64 V01
(* Tests LD1/ST1 single structure with 8 bit lanes *)
{
x=1;
0:X0=x; 0:X1=y;
}
 P0                 ;
 LD1 {V0.B}[2],[X0] ;
 ST1 {V0.B}[2],[X1] ;
forall(y=1)
