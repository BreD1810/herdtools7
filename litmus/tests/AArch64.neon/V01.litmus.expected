Test V01 Required
Histogram (1 states)
1000000:>y=1;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=1) is validated
Hash=031ceb20e75371542b14544aaf6473bb
Observation V01 Always 1000000 0
